proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -624,120 +619,120 @@ function constructorIsBuffer( test )
   test.identical( got, expected );
 
-  test.case = 'check U64x';
+  test.case = 'check U64x constructor';
   var got = _.constructorIsBuffer( U64x );
   var expected = true;
   test.identical( got, expected );
 
   /**/
 
-  test.case = 'check I8x';
+  test.case = 'check I8x constructor';
   var got = _.constructorIsBuffer( I8x );
   var expected = true;
   test.identical( got, expected );
 
-  test.case = 'check I16x';
+  test.case = 'check I16x constructor';
   var got = _.constructorIsBuffer( I16x );
   var expected = true;
   test.identical( got, expected );
 
-  test.case = 'check I32x';
+  test.case = 'check I32x constructor';
   var got = _.constructorIsBuffer( I32x );
   var expected = true;
   test.identical( got, expected );
 
-  test.case = 'check Ix';
+  test.case = 'check Ix constructor';
   var got = _.constructorIsBuffer( Ix );
   var expected = true;
   test.identical( got, expected );
 
-  test.case = 'check I64x';
+  test.case = 'check I64x constructor';
   var got = _.constructorIsBuffer( I64x );
   var expected = true;
   test.identical( got, expected );
 
   /**/
 
-  test.case = 'check F32x';
+  test.case = 'check F32x constructor';
   var got = _.constructorIsBuffer( F32x );
   var expected = true;
   test.identical( got, expected );
 
-  test.case = 'check Fx';
+  test.case = 'check Fx constructor';
   var got = _.constructorIsBuffer( Fx );
   var expected = true;
   test.identical( got, expected );
 
-  test.case = 'check F64x';
+  test.case = 'check F64x constructor';
   var got = _.constructorIsBuffer( F64x );
   var expected = true;
   test.identical( got, expected );
 
-  /**/
+  /* BufferNode */
 
   if (Config.interpreter === 'njs')
   {
-    test.case = 'check BufferNode';
+    test.case = 'check BufferNode constructor';
     var got = _.constructorIsBuffer( BufferNode );
     var expected = false;
     test.identical( got, expected );
 
-    test.case = 'check empty BufferNode';
+    test.case = 'check empty BufferNode instance';
     var got = _.constructorIsBuffer( BufferNode.alloc( 3 ) );
     var expected = false;
     test.identical( got, expected );
   }
 
-  /**/
+  /* BufferRaw */
 
-  test.case = 'check BufferRaw';
+  test.case = 'check BufferRaw constructor';
   var got = _.constructorIsBuffer( BufferRaw );
   var expected = false;
   test.identical( got, expected );
 
-  test.case = 'check empty BufferRaw';
+  test.case = 'check empty BufferRaw instance';
   var got = _.constructorIsBuffer( new BufferRaw( 3 ) );
   var expected = false;
   test.identical( got, expected );
 
-  test.case = 'check BufferRawShared';
+  test.case = 'check BufferRawShared constructor';
   var got = _.constructorIsBuffer( BufferRawShared );
   var expected = false;
   test.identical( got, expected );
 
-  /**/
+  /* BufferView */
 
-  test.case = 'check BufferView';
+  test.case = 'check BufferView constructor';
   var got = _.constructorIsBuffer( BufferView );
   var expected = false;
   test.identical( got, expected );
```

Comment:
Please, sort routines in order of file ./l0/l5/fBuffer.s:
```
  bufferRawIs,
  bufferTypedIs,
  bufferViewIs,
  bufferNodeIs,
  bufferAnyIs,
  bufferBytesIs,
  constructorIsBuffer,
```

Now, routines order is:
```
    bufferRawIs,
    bufferTypedIs,
    bufferNodeIs,
    bufferViewIs,
    bufferAnyIs,
    bufferBytesIs,
    constructorIsBuffer,
```

Reply:
Routines bufferNodeIs, bufferViewIs, bufferTypedIs, bufferRawIs have ```if( !Config.debug ) ``` condition for test cases without error throw, so should I remove this condition ?